spec/controllers/slides_controller_spec.rb

```diff
@@ -151,7 +151,7 @@
       login_by_user default_user
       post :update, params: { id: data.id, slide: data.attributes }
       expect(response.status).to eq(200)
-      expect(response).to render_template 'slides/aws/edit'
+      expect(response).to render_template "slides/#{CloudConfig::service_name}/edit"
```

Comment:
__[RuboCop]__

Style/ColonMethodCall: Do not use `::` for method calls.


<!-- Comment by [SideCI](https://sideci.com) -->